x/builder/keeper/keeper.go

```diff
@@ -8,22 +8,23 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	rewardsaddressprovider "github.com/skip-mev/pob/x/builder/rewards_address_provider"
```

Comment:
incredibly irrelevant nit: It seems common to abbreviate when importing long module names.